classes/controller/FrontController.php

```diff
@@ -896,6 +942,14 @@ public function setMedia()
             }
         }
 
+        if (Tools::isSubmit('live_edit') && Tools::getValue('ad') && Tools::getAdminToken('AdminModulesPositions'.(int)Tab::getIdFromClassName('AdminModulesPositions').(int)Tools::getValue('id_employee'))) {
```

Comment:
I see a lot of these tests about live edit, tokens etc. Please consider factorization (dedicated method(s) returning a boolean)

Reply:
I insist : please create one or more methods dedicated to checkings (submitted data, configuration values...). It will be easier to read and understand something like this :
```php
if ($this->isLiveEditAvailable() && $this->validateToken()) {
    $this->addJqueryUi('ui.sortable');
    /* etc. */
}
```
(please do it everywhere you see a whole bunch of `&&` in your `if` statements)